modules/web/server/src/main/resources/app.conf

```diff
@@ -35,7 +35,7 @@ web-server {
     external-base-url = "local.lucuma.xyz:8081"
 
     tls {
-        key-store = "modules/web/server/cacerts.jks.dev"
+        key-store = "cacerts.jks.dev"
```

Comment:
Unrelated, server seems to already resolve the path.

Reply:
Not on my workspace